src/tools/rbd_mirror/image_map/StateTransition.h

```diff
@@ -27,13 +29,21 @@ class StateTransition {
 
   struct Transition {
     State next_state;
```

Comment:
Nit: since you have a default constructor, you should default initialize all POD-types to avoid coverity false positives